### DISCUSSION TITLE
Access current style/class-values in JSX in order to append instead of overwriting

### DISCUSSION THREAD

**Discussion by klener89** [solidjs/solid#1576] [Q&A]
Hi,
I'm attempting to access the current Style/Class property in a JSX Element to update it conditionally without overwriting all non-matching elements

Specifically I want to set the Style of an element, if the index matches. Otherwise do nothing.

This examples overwrites the style if the element does not match. The counter gets incremented onMouseMove and all non-matching elements should not be updated.

```js
  const [counter, setCounter] = createSignal(0);
  const elements = [0, 1, 2, 3, 4, 5];

  return (
    <For each={elements}>
      {(element, index) => (
        <div style={counter() == index() ? `left:${counter()}px` : ``}>{element}</div>
      )}
    </For>
  );
```

Is there away to access the current value of style? Something like ``style={s => (counter() == index() ?  `left:${0}px;`+s : s)}``

**Comment by lxsmnsyc** [comment]
You should probably utilize createSelector to prevent all items from reacting to `counter` but only when it matches.
